README.md

```diff
@@ -74,3 +74,8 @@ In a terminal, execute:
 ```
 npm i
 ```
+
+# Information on data transmission during development
+
+When the extension sending telemetry is running in `development mode`, the data are sent to the `test.vscode` project.
```

Comment:
`, or whatever project bound to the optional [segmentWriteDebugKey](https://github.com/redhat-developer/vscode-commons/blob/master/INSTRUCTIONS.md#optional-add-a-custom-segment-key-in-packagejson-file)`